From f55f785820c432ddee07ca6ecb799f527a28a15f Mon Sep 17 00:00:00 2001
From: xzl <xiangzelong@deepin.org>
Date: Wed, 19 Apr 2023 14:11:44 +0800
Subject: [PATCH] fix: Remove the broken unit tests

---
 tests/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/meson.build b/tests/meson.build
index 6c1c461..df79f43 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -61,7 +61,6 @@ if cairo_dep.found()
 
     if host_system != 'darwin'
       tests += [
-        [ 'test-layout', [ 'test-layout.c', 'test-common.c' ], [ libpangocairo_dep, libpangoft2_dep ] ],
         [ 'test-fonts', [ 'test-fonts.c', 'test-common.c' ], [ libpangocairo_dep, libpangoft2_dep ] ],
         [ 'test-no-fonts', [ 'test-no-fonts.c' ], [ libpangocairo_dep, libpangoft2_dep ] ],
       ]
-- 
2.33.1

